package.json

```diff
@@ -43,7 +43,7 @@
     "webpack-log": "^1.2.0"
   },
   "peerDependencies": {
-    "typescript": "^2.7"
+    "typescript": "^3"
```

Comment:
I think you might want `^2.7 | ^3` (or whatever the syntax is), since it would still be totally valid to use TS 2.7 with this lib

Reply:
Updated.